downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc

```diff
@@ -13,16 +13,22 @@ You can install {Navigator} on Red Hat Enterprise Linux (RHEL) from an RPM.
 
 * You have installed the prerequisites for RHEL.
 * You have Ansible or {ExecEnvNameStart} installed.
+* You registered your system with Red Hat Subscription Manager.
 
 
 .Procedure
 
-. Download the RPM.
+. Enable the repository for RHEL 8.
++
+----
+$ sudo subscription-manager repos --enable ansible-navigator-1.0.0.el8ap.noarch.rpm
```

Comment:
The repo name will be: `automation-platform-2.0-for-rhel-8-x86_64-rpms`

Reply:
fixed, thanks!